documentation/dev-portal/src/nymvpn/cli-linux.md

```diff
@@ -0,0 +1,77 @@
+# NymVPN alpha CLI: Guide for GNU/Linux
+
+```admonish warning
+NymVPN is an experimental software and it's for [testing](testing.md) purposes only. All users testing the client are expected to sign GDPR Information Sheet and Consent Form (shared at the event), and follow the steps listed in the form [*NymVPN User research*](https://opnform.com/forms/nymvpn-user-research-at-37c3-yccqko-2).
```

Comment:
are we continuing to use the same form as we did @ 37c3? or is this an old link

Reply:
This was for Neuchatel (`-2`), I will upgrade it for any next event I guess.